File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int,int> count;
        int res=0,maxcount=0;
        for(int num:nums){
            count[num]++;
            if(count[num]>maxcount){
                res=num;
                maxcount=count[num];
            }
        }
        return res;
        
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {
        ArrayList<Integer>res=new ArrayList<>();
        int l=0;
        res.add(nums[l]);
        int r=l+1;
        while(r<nums.length){
            if(nums[l]==nums[r]){
                r++;
            }else{
                res.add(nums[r]);
                l=r;
                r++;

            }
        }
        for(int i=0;i<res.size();i++){
            nums[i]=res.get(i);
        }
        return res.size();
        
    }
}